packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/ScatterPlotViewer/helpers/utils.spec.js

```diff
@@ -113,4 +133,197 @@ describe('ScatterPlotViewer > helpers > utils', function () {
       })
     })
   })
+
+  describe('getDataPoints utility function', function () {
+    describe('when there is a single series of data', function () {
+      it('should return an array of a seriesData object containing an array of x, y data points', function () {
+        const points = getDataPoints(randomSingleSeriesData.data)
+        expect(points).to.be.an('array')
+        expect(points[0].seriesData).to.have.lengthOf(randomSingleSeriesData.data.x.length)
+        points[0].seriesData.forEach((point) => {
+          expect(point.x).to.be.a('number')
+          expect(point.y).to.be.a('number')
+        })
+      })
+    })
+
+    describe('when there is multiple series of data', function () {
+      it('should return the same data JSON object input as the function parameter', function () {
+        const inputData = lightCurveMockData.variableStar.data
+        const points = getDataPoints(inputData)
+        expect(points).to.deep.equal(inputData)
```

Comment:
What is the purpose of this function (`getDataPoints`)? It seems a bit odd to feed data into a function, and expect the return to be the exact same as the data that was sent in.

Reply:
The `ScatterPlotViewer` supports two JSON formats: one type that only can be for a single series of data aka TESS's JSON format and one type that supports either single or multiple data series (The goal is to eventually migrate the TESS subject viewer to use the more general `ScatterPlotViewer` since it is tested and the original d3 LCV mostly is not). The utility function here converts the first type (TESS-like single data series) to be like the latter type (can be multiple data series), but if the data from the subject JSON file is already in that structural format, then just return it as is. 

I'll plan on adding some info about this into the scatter plot readme to clarify what's going on here in a follow up PR. Thanks for the question.